vars/common-vars.yml

```diff
@@ -31,7 +31,18 @@ socok8s_repos_to_configure:
   openSUSE-Leap-15.0-Update: "{{ obs_mirror }}/update/leap/15.0/oss/"
   openSUSE-Leap-15.0-Cloud: "{{ obs_mirror }}/repositories/Cloud:/Tools/openSUSE_Leap_15.0/"
   Leap15develtools: "{{ obs_mirror }}/repositories/devel:/tools/openSUSE_Leap_15.0/"
+  SLE-15-SP1-JeOS-GM-Full: http://download.suse.de/full/full-sle15-sp1-x86_64/
+  SLE-15-SP1-JeOS-GM-Update: http://download.suse.de/ibs/SUSE:/SLE-15-SP1:/Update/standard/
+  # needed for python-barbicanclient
+  SLE-15-SP1-JeOS-GM-backports: "https://download.opensuse.org/repositories/openSUSE:/Backports:/SLE-15-SP1/standard/"
+  # needed for ruby2.5-devel
+  SLE-15-SP1-JeOS-GM-ruby: "https://download.opensuse.org/repositories/devel:/languages:/ruby/SLE_15_SP1/"
+  # needed for ruby2.5-rubygem-bundler
+  SLE-15-SP1-JeOS-GM-rubyextensions: "https://download.opensuse.org/repositories/devel:languages:ruby:extensions/SLE_15_SP1/"
+  # phantomjs
+  SLE-15-SP1-JeOS-GM-openstack-master: "https://download.opensuse.org/repositories/Cloud:/OpenStack:/Master/SLE_15_SP1/"
```

Comment:
ugh, that doesn't sound like a stable solution..

Reply:
Agreed.